frontend/animal/EndpointTable.js

```diff
@@ -32,11 +32,18 @@ class EndpointTable {
     }
 
     hasValues(val) {
+        return _.chain(this.endpoint.data.groups)
+            .map(d => _.isNumber(d[val]))
+            .some()
+            .value();
+    }
+
+    hasData(val) {
         return _.chain(this.endpoint.data.groups)
             .map(function(d) {
                 return d[val];
             })
-            .some($.isNumeric)
+            .some(x => x != null)
```

Comment:
@casey1173 curious - why did this line change?

Reply:
can we just delete `hasData` and use `hasValues` everywhere?

Reply:
hasValues checks if all string values are numeric, and treatment related effect values are 'yes' or 'no'

Reply:
can we change to use `!_.isNil` and get rid of the duplicative function? https://lodash.com/docs/4.17.15#isNil